src/cli.zig

```diff
@@ -1054,17 +1054,15 @@ pub const Arguments = struct {
                 ctx.debug.silent = true;
             }
 
-            if (opts.define) |define| {
-                if (define.keys.len > 0)
-                    bun.JSC.RuntimeTranspilerCache.is_disabled = true;
-            }
-        }
-
-        if (cmd == .RunCommand or cmd == .AutoCommand or cmd == .BunxCommand) {
```

Comment:
this is because `BunxCommand` doesn't run `Arguments.parse`, so may as well simplify.